### ISSUE TITLE
Namespace for protocol version configmap is hardcoded

### ISSUE THREAD

**Issue by skonto** [lightbend/cloudflow#6]
**Describe the bug**
The namespace for the protocol version configmap cloudflow-protocol-version is hardcoded to lightbend.
If the lightbend namespace doesn't exist, the configmap is not created, and the CLI cannot be used.
It should not be hardcoded.

**To Reproduce**
Install Cloudflow on a namespace different than lighbend (can be changed in [install-gke.sh](https://github.com/lightbend/cloudflow-installer/blob/ff701f9e4aa313f55712d7fabba16c44eec852be/install-gke.sh#L36)).

Run kubectl cloudflow list.

It fails with
```
[Error] Cannot find the 'cloudflow-protocol-version' configmap, please make sure that the Cloudflow operator is installed
```
***workaround***: create a `lightbend` namespace, and kill the `cloudflow-operator` pod to restart it.

**Expected behavior**
The configmap to be installed and looked for in the current namespace.

**Screenshots**
N/A
**Additional context**
N/A

Thank you @skyluc for reporting this.

**Comment by mrooding** [comment]
This can be closed since #45 was merged?

**Comment by RayRoestenburg** [comment]
I agree, this can be closed. @skonto @debasishg
